st2-ask/notification_rule.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: "notify_slack_execution_change"
+pack: "st2"
+description: "Watches for an execution status of "awaits" for a certain action ref
+enabled: true
+
+trigger:
+  type: core.st2.generic.actiontrigger
+
+criteria:
+  trigger.action_ref:
+      type: equals
+      pattern : st2.ask
+  trigger.status:
+      type: equals
+      pattern : awaits
+
+action:
+  ref: slack.post_message
+  parameters:
+    message: '{{trigger.execution_id}} is awaiting an approval action'
```

Comment:
> Which channel is this going to post to? Can this be a direct message?

The reason I'm asking is I don't see where the channel, email, or target audiences are passed in as data into this rule. The rule criteria is so generic. This means that any st2.ask in pending state will trigger this notification to the same set of individuals. What if I want specific st2.ask to ask 1 set of individuals while another st2.ask to ask another set of individuals?